SampleGallery/Samples/SDK 17134/RadialGradients/RadialGradients.xaml.cs

```diff
@@ -0,0 +1,310 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. 
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, 
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH 
+// THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+//*********************************************************
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI;
+using Windows.UI.Composition;
+using System.Numerics;
+using System.Diagnostics;
+
+namespace CompositionSampleGallery
+{
+    public sealed partial class RadialGradients : SamplePage
+    {
+        public RadialGradients()
+        {
+            this.InitializeComponent();
+        }
+
+        public static string StaticSampleName => "Radial Gradients";
+        public override string SampleName => StaticSampleName;
+        public static string StaticSampleDescription => "Sample application for radial gradient brush";
+        public override string SampleDescription => StaticSampleDescription;
+
+
+        private Compositor _compositor;
+
+        private SpriteVisual _buttonVisual;
+        private SpriteVisual _pulseVisual;
+
+        private CompositionRadialGradientBrush _buttonBrush;
+        private CompositionRadialGradientBrush _pulseBrush;
+
+        private CompositionColorGradientStop _BBGradientStop1;
+        private CompositionColorGradientStop _BBGradientStop2;
+
+        private CompositionColorGradientStop _PBGradientStop1;
+        private CompositionColorGradientStop _PBGradientStop2;
+        
+        private ScalarKeyFrameAnimation _stop1OffsetAnim;
+        private ScalarKeyFrameAnimation _stop2offsetAnim2;
+        private ColorKeyFrameAnimation _pulseColor;
+        private Vector3KeyFrameAnimation _scale;
+
+        private ColorKeyFrameAnimation _changeButtonGradientStop1;
+        private ColorKeyFrameAnimation _changeButtonGradientStop2;
+
+        private Color _blueViolet;
+        private Color _lightBlue;
+        private Color _indianRed;
+        private Color _orangeRed;
+        private Color _transparent;
+        private Color _aliceBlue;
+
+        private Boolean animationOn;
```

Comment:
Prefix member with _

Reply:
And use the alias "bool" for the type rather than "Boolean".

Reply:
updated